src/App.jsx

```diff
@@ -14,57 +15,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [isGood, setGood] = useState('Jam');
+  const [isActive, setIsActive] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const handleClick = good => {
+    if (isActive === good) {
+      setGood('');
+      setIsActive(null);
+    } else {
+      setGood(good);
+      setIsActive(good);
+    }
+  };
```

Comment:
Here you just setting two completely identical states, what's the point of that? Could be just:
```jsx
  const [selectedGood, setSelectedGood] = useState(null) // no initial good selected

  const handleClick = good => {
    if (selectedGood === good) {
      setSelectedGood(null) // click on the selected good resets the state
    } else {
      setSelectedGood(good) // set new good otherwise
    }
  };
```